.github/workflows/dependabot-automerge.yml

```
@@ -0,0 +1,38 @@
+        if: |
+          steps.metadata.outputs.update-type == 'version-update:semver-minor' || 
+          steps.metadata.outputs.update-type == 'version-update:semver-patch' ||
+          steps.metadata.outputs.package-ecosystem == 'gitsubmodule'
+        run: gh pr review --approve "$PR_URL"
+        env:
+          PR_URL: ${{ github.event.pull_request.html_url }}
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Enable auto-merge for minor, patch, and submodule updates
+        if: |
+          steps.metadata.outputs.update-type == 'version-update:semver-minor' || 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)